plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -3636,4 +3703,9 @@ public void commitTransaction(long transactionId)
     {
         delegate.commitTransaction(transactionId);
     }
+
+    private static boolean isLocationPresent(Storage storage)
+    {
+        return storage.getOptionalLocation().isPresent() && isNotEmpty(storage.getLocation());
```

Comment:
Why do we need `isNotEmpty` here?

cc @losipiuk i remember we had a conversation about that

Reply:
When dealing with views, their storage location will be an empty string

Reply:
Let's have a code comment.